server/src/app.ts

```diff
@@ -37,6 +38,7 @@ app.use("/user", userRoutes);
 app.use("/projects", isAuthenticated, projectRoutes);
 app.use("/categories", isAuthenticated, categoryRoutes);
 app.use("/tablegroups", isAuthenticated, tableGroupRoutes);
+app.use("/criteria", isAuthenticated, criteriaRoutes);
```

Comment:
Typically, routes are always in the plural form, so can you make the route `/criterias`?

Reply:
criteria is the plural, criterion is the singular I believe